boardcfg/number1/common_settings.h

```diff
@@ -40,24 +48,47 @@
 #define USB_POWER_CONSUMPTION 100
 
 /**
- * @brief Use a development UID rather than reading the UID from flash.
+ * @}
  *
- * Intended for development use only. This can be used to avoid linker errors
- * when running without a bootloader.
+ * @name RDM Identification
+ * Testing
+ * @{
+ */
+
+/**
+ * @brief The RDM manufacturer ID
+ */
+#define CFG_MANUFACTURER_ID 0x7a70
```

Comment:
Out of interest, are the model numbers going to match in Stellascapes land? I guess that's a question for @mrpackethead ?